src/components/access-type.js

```diff
@@ -29,36 +29,39 @@ class AccessTypeBase extends React.Component {
     if (!enableAccesstype) {
       return false;
     }
-
-    if (accessTypeKey && !global.AccessType && global.document) {
+    const HOST = isStaging ? staging_Host : prod_Host;
+    const environment = isStaging ? "&env=sandbox" : "";
+    const accessTypeHost = `${HOST}/frontend/v2/accesstype.js?key=${accessTypeKey}${environment}`;
+    const isATScriptAlreadyPresent = document.querySelector(`script[src="${accessTypeHost}"]`);
+    if (accessTypeKey && !isATScriptAlreadyPresent && !global.AccessType && global.document) {
       const accessTypeScript = document.createElement("script");
-      const HOST = isStaging ? staging_Host : prod_Host;
-      const environment = isStaging ? "&env=sandbox" : "";
-
-      const accessTypeHost = `${HOST}/frontend/v2/accesstype.js?key=${accessTypeKey}${environment}`;
-
       accessTypeScript.setAttribute("src", accessTypeHost);
+      accessTypeScript.setAttribute("id", 'AccessTypeScript');
       accessTypeScript.setAttribute("data-accessType-script", "1");
       accessTypeScript.async = 1;
       accessTypeScript.onload = () => callback();
       document.body.appendChild(accessTypeScript);
       return true;
     }
-
     global.AccessType && callback();
     return true;
   };
 
-  setUser = async (emailAddress, mobileNumber, accesstypeJwt) => {
+  setUser = async (emailAddress, mobileNumber, accesstypeJwt, isLoggedIn = true) => {
     if (!global.AccessType) {
-      return {};
+      return null;
+    }
```

Comment:
`setUser` should return a falsy value if user is not present as we're using the returned value to call other APIs.

Reply:
But here you are checking for `global.AccessType`, and not user.

Reply:
I meant if I return an empty Object we've a check like `if(user)` which will break, right?

Reply:
here. https://github.com/quintype/quintype-node-components/pull/140/files#diff-b92dc5c18814b82a081cd404093cdf50R171